docs/source/background/channels.md

```diff
@@ -0,0 +1,88 @@
+## Channels
+
+
+Channels are asynchronous messaging queue used to move messages between [Nodes](./nodes.md) in the RapidsMPF streaming network.  
+
+<div style="display: flex; align-items: flex-start; gap: 24px;">
+  <img src="../_static/buffers-animated.gif" alt="Animated buffer pipeline" style="max-width: 4500px;"/>
+  <img src="../_static/animation-legend.png" alt="Animation Legend" style="width: 320px;"/>
+</div>
+
+As buffers move through the graph, the channels (arrows) move from empty (dashed line) to full (solid line).   
+
+*Note: Nodes do not refer to number of GPUs or number of machines -- nodes and edges define a graph for RAPIDSMPF to execute in either Single or Multiple GPU*
+
+```
+┌─────────────────────────────────────────────────────────────────────────┐
+│                          STREAMING NETWORK                              │
+│                                                                         │
+│  ┌──────────┐         ┌──────────┐         ┌──────────┐                 │
+│  │  Node 1  │ ──ch1─> │  Node 2  │ ──ch2─> │  Node 3  │                 │
+│  │(Producer)│         │(Transform)         │(Consumer)│                 │
+│  └──────────┘         └──────────┘         └──────────┘                 │
+│       │                    │                     │                      │
+│    Message             Message               Message                    │
+│                                                                         │
+└─────────────────────────────────────────────────────────────────────────┘
+```
+*fig: example streaming network with 3 Nodes and 2 Channels*
+
+Components:
+  • Node: Coroutine that processes messages
+  • Channel: Async queue connecting nodes
+  • Message: GPU Buffer with a CUDA Stream
+
+In the above graph, moving data in and out of channels on a single GPU should be relatively cheap, nearly free! This stratedy of using channels to move tasks/buffers is a core methodology for RapidsMPF to overlap: scans, compute, spilling, and communication.
```

Comment:
In particular, I'm curious whether sending to / receiving from a channel will ever

1. Serialize / deserialize data
2. Copy data (either from host to device or device to host, or on a device)
3. Spill / unspill data